assets/context.py

```diff
@@ -0,0 +1,187 @@
+import asyncio
+from collections import namedtuple
+
+import discord
+from discord.ext import commands
+
+from cogs.utils import misc, check
+from cogs.utils.embeds import ConfirmEmbed, ConfirmEmbedAnother
+
+argument = namedtuple("Usage", ["required", "text"])
+example = namedtuple("Examples", ["args", "text"])
+sub_desc = namedtuple("subcommands", "desc")
+others = namedtuple("others", ["name", "text"])
+usage_info = namedtuple("usage_information", ["command", "value"])
+keyword = namedtuple("keywords", ["keyword", "desc"])
```

Comment:
C0103: Class name doesn't conform to PascalCase naming style

translation : Capitalize that